hid-replay.py

```diff
@@ -108,15 +108,17 @@ def inject_events(self, wait_max_seconds=2):
 def main(argv):
     try:
         replay = HIDReplay(argv[0])
-        while uhid.UHIDDevice.process_one_event(1000):
+        try:
+            while uhid.UHIDDevice.process_one_event(1000):
+                pass
+            print('Hit enter (re)start replaying the events')
+            sys.stdin.readline()
+            replay.inject_events()
+        except KeyboardInterrupt:
             pass
-        print('Hit enter (re)start replaying the events')
-        sys.stdin.readline()
-        replay.inject_events()
-    except KeyboardInterrupt:
-        pass
-    finally:
         replay.destroy()
+    finally:
+        pass
```

Comment:
Is this really reasonable? :)

Reply:
we have to contribute to global warming somehow. a few spare cpu cycles was all I could think of.